content/en/docs/tasks/security/authz-http/index.md

```diff
@@ -32,178 +30,243 @@ the product page, you can see the following sections:
-"ratings" service. To fix this issue, you need to grant the `reviews` service access to the `ratings` service.
-We will show how to do that in the next step.
-
-{{< tip >}}
-There may be some delays due to caching and other propagation overhead.
-{{< /tip >}}
-
-### Step 3. allowing access to the `ratings` service
-
-We will create a policy to allow the `reviews` service to access the `ratings` service. Note that in the
-[setup step](#before-you-begin), we created a `bookinfo-reviews` service account for the `reviews` service. This
-service account is the authenticated identify for the `reviews` service.
-
-Run the following command to create a policy that allows the `reviews` service to access the `ratings` service.
-
-{{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_service_level_access_control_step3_apply.sh" >}}
-
-Once applied, the policy has the following effects:
-
-*   Creates a `ServiceRole` `ratings-viewer` which allows access to the `ratings` service.
-
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_service_level_access_control_step3_ratings_viewer.yaml" >}}
-
-*   Creates a `ServiceRoleBinding` `bind-ratings` which assigns `ratings-viewer` role to the
-`cluster.local/ns/default/sa/bookinfo-reviews` service account, which represents the `reviews` service.
-
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_service_level_access_control_step3_bind_ratings.yaml" >}}
-
-Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`). Now you should see
-the "black" and "red" ratings in the "Book Reviews" section.
-
-{{< tip >}}
-There may be some delays due to caching and other propagation overhead.
-{{< /tip >}}
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: "security.istio.io/v1beta1"
+kind: "AuthorizationPolicy"
+metadata:
+  name: "bookinfo-viewer"
+  namespace: default
+spec:
+  rules:
+  - from:
+    - source:
+        namespaces: ["default", "istio-system"]
+    to:
+    - operation:
+        methods: ["GET"]
+EOF
+{{< /text >}}
 
-## Cleanup
+You can expect to see output similar to the following:
 
-*   Remove Istio authorization policy configuration:
+{{< text plain >}}
+authorizationpolicy.security.istio.io/bookinfo-viewer created
+{{< /text >}}
 
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="remove_istio_authorization_policy.sh" >}}
+Now if you point your browser at Bookinfo's `productpage` (`http://$GATEWAY_URL/productpage`).
+You should see the "Bookinfo Sample" page, with the "Book Details" section in the lower left part
+and the "Book Reviews" section in the lower right part.
 
-    Alternatively, you can delete all `ServiceRole` and `ServiceRoleBinding` resources by running the following commands:
+### Disable namespace-level access control
```

Comment:
Same here, "clean up" make more sense to me.

Reply:
+1

Reply:
I made the change from @rcaballeromx's comment, I can revert it back but note we cannot have multiple `### Clean up`, we have to make the title different from each other.